llama-index-integrations/llms/llama-index-llms-gemini/llama_index/llms/gemini/base.py

```diff
@@ -33,21 +33,25 @@
 
 
 GEMINI_MODELS = (
-    "models/gemini-pro-vision",
-    "models/gemini-pro-vision-latest",
-    "models/gemini-ultra-vision",
-    "models/gemini-ultra-vision-latest",
+    # Gemini 1.0 Pro Vision has been deprecated on July 12, 2024.
+    # According to official recommendations, switch the default model to gemini-1.5-flash
+    "models/gemini-1.5-flash",
+    "models/gemini-1.5-flash-latest",
+    # "models/gemini-pro-vision",
```

Comment:
Can we remove the lines corresponding to the unsupported models rather than commenting them out?

Reply:
Of course, the reason for the annotation is personal habit.